agent/test/event/Example.java

```diff
@@ -0,0 +1,11 @@
+public class Example {
```

Comment:
I think it would be better for this to be in the `com.appland.appmap.test.fixture` package.